kuma/wiki/tests/test_views.py

```diff
@@ -1190,6 +1190,22 @@ def test_create_on_404(self):
                                            locale=locale))
         eq_(302, response.status_code)
 
+    def test_creating_child_of_redirect(self):
+        """While try to create a child of a redirect,
+        the parent of the child should be redirect's parent"""
+        self.client.login(username='admin', password='testpass')
+        rev = revision(is_approved=True, save=True)
+        doc = rev.document
+        child_full_slug = doc.slug + "/" + "children_document"
+        doc._move_tree(new_slug="moved_doc")
```

Comment:
``_move_tree`` is such a complex way to create the redirect.  Manually creating the two pages would be faster.  A redirect is just a page where the content looks like ``REDIRECT_CONTENT``.